internal/api/v1/services/instance.go

```diff
@@ -44,18 +44,20 @@ func (s *InstanceService) ListInstances(ctx context.Context, opts *models.ListOp
 // CreateInstance creates a new instance
 func (s *InstanceService) CreateInstance(
 	ctx context.Context,
-	name,
+	instanceName,
 	projectName,
 	webhookURL string,
 	instances []infrastructure.InstanceRequest,
 ) (*models.Job, error) {
 	// Create job request
+	jobName := fmt.Sprintf("%s-%s-creation", projectName, instanceName)
 	jobReq := &infrastructure.JobRequest{
-		Name:        name,
-		ProjectName: projectName,
-		Provider:    instances[0].Provider,
-		Instances:   instances,
-		Action:      "create",
+		JobName:      jobName,
+		InstanceName: instanceName,
+		ProjectName:  projectName,
+		Provider:     instances[0].Provider,
```

Comment:
I always had a question about the providers. if user defines multiple providers for various instance groups, why do we use only the first one for the job? we don't even need a provider for a job! we can just use provider objects when provisioning instances.